box-ui/public/manifest.json

```diff
@@ -1,25 +1,31 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "JBox",
+  "name": "JitsiBox PWA",
   "icons": [
     {
       "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
+      "sizes": "64x64",
+      "type": "image/x-icon",
+      "purpose" : "maskable"
     },
     {
       "src": "logo192.png",
       "type": "image/png",
-      "sizes": "192x192"
+      "sizes": "192x192",
+      "purpose" : "maskable any"
     },
     {
       "src": "logo512.png",
       "type": "image/png",
-      "sizes": "512x512"
+      "sizes": "512x512",
+      "purpose" : "maskable any"
     }
   ],
-  "start_url": ".",
+  "dir": "ltr",
+  "start_url": "/",
```

Comment:
Is it necessary to change the start url to `/` ? Won't it make the url appear as `https://example.com/` instead of `https://example.com` ?

Reply:
Not necessary. The PWA isn't completed.